laa_court_data_api_app/routers/hearing_summaries.py

```diff
@@ -33,7 +33,8 @@ async def get_hearing_summaries(urn: str):
             prosecution_case_results = ProsecutionCasesResults(**cda_response.json())
             summaries = map_hearing_summaries(prosecution_case_results.results)
             logging.info(f"Hearing_Summaries_To_Show: {summaries.count}")
-            return HearingSummariesResponse(hearing_summaries=summaries,
+            return HearingSummariesResponse(prosecution_case_reference=urn,
```

Comment:
Is this how you want to be doing it, or should there be a prosecution_case_reference per summary?